sample/DokanNetMirror/Mirror.cs

```diff
@@ -380,7 +380,18 @@ public NtStatus SetFileAttributes(string fileName, FileAttributes attributes, Do
         {
             try
             {
+                var stream = info.Context as FileStream;
+                if (stream != null)

```

Comment:
```C#
var stream = info.Context as FileStream;
if (stream != null)
```
can be replaced width this in [C#7](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7#pattern-matching)
```C#
if (info.Context is FileStream stream)
```